src/main/java/com/algorand/algosdk/v2/client/common/Client.java

```diff
@@ -48,7 +48,7 @@ public static HttpUrl getHttpUrl(QueryData qData, int port, String host) {
         httpUrlBuilder.port(port);
 
         for (String ps : qData.pathSegments) {
-            httpUrlBuilder.addPathSegment(ps);
+            httpUrlBuilder.addEncodedPathSegment(ps);
```

Comment:
@algochoi A few remarks - Let me know how you'd like to proceed:
* The change is _not_ auto-generated.
* It's a holdover from #336 when the encoded Box name was provided via path segment.
* Prior to your note, I briefly considered reverting and opted to leave as is.  I don't feel strongly.  
  * If you have a strong preference, I can be convinced otherwise.
  * Rationale for keeping as is:  The as is behavior is arguably broken.  The entire URL is subject to URL encoding.
  * Rationale for revert:  It's unrelated to the PR's problem space.

Reply:
Got it, I was wondering because it was included in this commit: `generated code`
https://github.com/algorand/java-algorand-sdk/pull/336/commits/0018a7382fe2b9265cbd8fe29de28ffe222b3e10#diff-bed26bf8419f0865a29ec692e2e32b08fe01e1059265992b62ff546b76574a61R51

I think it is fine as is then, resolving